api/src/pcapi/scripts/pro/populate_is_open_to_public_from_is_permanent.py

```diff
@@ -0,0 +1,10 @@
+from pcapi.core.offerers import models as offerer_models
+from pcapi.models import db
+
+
+def populate_venue_is_open_to_public_from_is_permanent() -> None:
+    venues = offerer_models.Venue.query.filter(offerer_models.Venue.isPermanent == True).all()
+
+    for venue in venues:
+        venue.isOpenToPublic = True
+        db.session.commit()
```

Comment:
Un commit par lieu n'est peut-être pas nécessaire :)
Que dis-tu d'une autre approche, plus directe ?

```suggestion
    offerer_models.Venue.query.filter(offerer_models.Venue.isPermanent == True).update({"isOpenToPublic": True})
```

Reply:
je trouve ça pas mal, 
mais comme je fais ces tickets aussi pour apprendre je vais essayer la méthode par batch que me propose olivier :)